lib/tom_queue/deferred_work_manager.rb

```diff
@@ -80,13 +80,19 @@ def schedule(response, headers, payload)
     def start
       debug "[DeferredWorkManager] Deferred process starting up"
 
+      register_signal_handlers
```

Comment:
No - for now we'll keep it the same as the DJ processes.

I'm trying not to be a pedant here, but a daemon is just a long running process that responds to requests. My point was it's not a CLI since typically you don't interact with it via the cli - a service manager starts the process and it works in the background.

Really, all we need right now is the `rescue SignalException` and not report to HB, though!